Controller/ContentTypeController.php

```diff
@@ -266,15 +298,15 @@ public function createContentTypeAction(Request $request, $contentTypeGroupId, $
 
     public function updateContentTypeAction(Request $request, $contentTypeId, $languageCode = null)
     {
-        $languageCode = $languageCode ?: $this->prioritizedLanguages[0];
+        $contentType = $this->contentTypeService->loadContentType($contentTypeId);
+        $languageCode = $this->getVerifiedContentTypeLanguageCode($contentType, $languageCode);
```

Comment:
hmm, API wise the language might be part of the draft but not the previous published version, so might not be the way to do it.